src/functions-and-arrays.js

```diff
@@ -52,14 +140,52 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUniqueArray) {
+let nuevoUniqueArray = wordsUniqueArray;
+  if(wordsUniqueArray.length == 0){
+    return null;
+  }else {
+    wordsUniqueArray.forEach((element, indice) => {
+      for(x = indice + 1; x < wordsUniqueArray.length; x++ ){
+          if(element === wordsUniqueArray[x]){
+            nuevoUniqueArray.splice(x,1);
+          }
+      }
+    })
+    return nuevoUniqueArray;
+  }
```

Comment:
aca podria hacer buscar o preguntar asi la palarba esta en el arreglo 

`  for (let word of words) {
    if (!uniqueArr.includes(word)) uniqueArr.push(word);
  }
  return uniqueArr;`